microshift_release_notes/microshift-4-18-release-notes.adoc

```diff
@@ -170,7 +170,11 @@ Issued: 1 May 2025
 
 {product-title} release 4.18.11 is now available, see the link:https://access.redhat.com/errata/RHBA-2025:4249[RHBA-2025:4249] advisory. Release notes for bug fixes and enhancements are provided in this documentation. The images that are included in the update are provided by the {OCP} link:https://access.redhat.com/errata/RHSA-2025:4211[RHSA-2025:4211] advisory.
 
-See the latest images included with {microshift-short} by xref:../microshift_updating/microshift-list-update-contents.adoc#microshift-get-rpm-release-info_microshift-list-update-contents[listing the contents of the {microshift-short} RPM release package].
+See the latest images included with {microshift-short} by using the following instructions:
+
+* xref:../microshift_updating/microshift-list-update-contents.adoc#microshift-get-rpm-release-info_microshift-list-update-contents[Listing the contents of the {microshift-short} RPM release package]
```

Comment:
Why is it called "listing" as opposed to bootc "getting"?
Getting might be more appropriate in this case, isn't it?

Reply:
@ggiguash That was previously approved content and is not being changed with this PR.

Reply:
In this case, I've no further comments.
LGTM

Reply:
Thank you!